examples/exclude-words/create_wordlist.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import glob
+import json
+import pathlib
+from pdfminer.high_level import extract_text
+
+
+def main(args):
+    file_path = args.file_path
+    frequency = int(args.minimal_frequency)
+    minimal_length = int(args.minimal_length)
+    words_file = args.words_file
+
+    if not file_path[-1] == '/':
+        file_path += '/'
+
+    word_dict = {}
+    for file in glob.glob(file_path + '*.pdf'):
+        text = extract_text(file)
+        sentence_list = text.split('\n')
+        sentence_list = [word for word in sentence_list if word != '']
+
+        for sentence in sentence_list:
+            for word in sentence.split(' '):
+                word = word.lower()
+                if len(word) > 1 and word[-1] == ':':
+                    word = word[:-1]
+                if not word.isnumeric() and len(word) >= minimal_length:
```

Comment:
Do we really want to skip all numeric strings? Maybe it could be an option to the program instead?